.rustfmt.toml

```diff
@@ -1,10 +1,27 @@
+# Configures Rust code formatting for Oak.
 # See https://github.com/rust-lang/rustfmt/blob/master/Configurations.md
-comment_width = 100
+#
+# Google rustfmt configuration.
+
+edition = "2021"
+version = "Two"
+
+# The "Default" setting has a heuristic which splits lines too aggresively.
+# We are willing to revisit this setting in future versions of rustfmt.
+# Bugs:
+#   * https://github.com/rust-lang/rustfmt/issues/3119
+#   * https://github.com/rust-lang/rustfmt/issues/3120
+use_small_heuristics = "Max"
+newline_style = "Unix"
+wrap_comments = true
+format_generated_files = false
+
+#
+# Overrides from Oak.
+#
+
```

Comment:
Could you set comment width and max width explicitly anyways? We don't want things to change when the default changes

Reply:
I clearly marked these as default value to avoid confusion. Also brought in all files now.